### ISSUE TITLE
Trailing slash in folder name causes validation error on Windows 

### ISSUE THREAD

**Issue by didiladi** [dynatrace-oss/dynatrace-monitoring-as-code#93]
**Describe the bug**
Running `monaco` on windows fails when adding a trailing slash to the folder name.

E.g. this works:

```
.\monaco-windows-4.0-amd64.exe -v --dry-run -se="dev-mc" -p="webhook" -e="my-environments.yaml" projects
```

Whereas this doesn't:

```
.\monaco-windows-4.0-amd64.exe -v --dry-run -se="dev-mc" -p="webhook" -e="my-environments.yaml" projects/
```

**How to reproduce**
Just run monaco on Windows

**Expected behavior**
The execution of `monaco` should work.

**Log output**
The error message:

```
2020-12-02T11:14:08.2179337Z 2020-12-02 11:14:08 INFO Processing environment Sandbox...
2020-12-02T11:14:08.2180244Z 2020-12-02 11:14:08 INFO Processing project projects\webhook...
2020-12-02T11:14:08.2181059Z 2020-12-02 11:14:08 DEBUG Deploying configs in this order:
2020-12-02T11:14:08.2182317Z 2020-12-02 11:14:08 DEBUG 1: projects\webhook\notification\notification-webhook.json
2020-12-02T11:14:08.2183282Z 2020-12-02 11:14:08 DEBUG Validating config projects\webhook\notification\notification-webhook.json
2020-12-02T11:14:08.2184313Z 2020-12-02 11:14:08 ERROR Validation of Sandbox failed with error Id 'newenvironment\alerting-profile\Sandbox_APP-0000' was not available. Please make sure the reference exists.
```

**Environment (please complete the following information):**
 - OS: Windows
 - Monaco version: 1.0.1

**Additional context**
This bug was found by @JACONDE in #80.

**Comment by baumgarb** [comment]
@didiladi I think I solved it. So far we made sure that the provided folder will end with the OS-specific path separator. That worked, but if you provide "projects/" then our previous logic just added the backslash on Windows which resulted in "projects/\". That should be fixed now.

I thought about refactoring this whole parsing of command line args and moving both the `readPath` and the `parseInputCommand` functions from `main.go` to `pkg/util/parse.go`. But I'm not "bold" enough to do that because the `parseInputCommand` function depends on the `environment` package which in turn depends on the `util` package. And I feel like I'm not yet familiar enough with this repo to refactor this circular dependency properly. Hope that's ok, but I guess this could be a separate improvement / refactoring ticket.
